### DISCUSSION TITLE
Highlight.js v11 is missing from CDNJS

### DISCUSSION THREAD

**Discussion by LunarTwilight** [cdnjs/cdnjs#14052] [General] [answered]
CDNJS seems to be missing highlight.js v11 despite it being released for 4 days now and highlight.js' readme pointing to it.

**Comment by MattIPv4** [comment, marked as answer]
👋 We have ongoing maintenance with the auto-update functionality of cdnjs currently -- this version was released since the maintenance started, and so our auto-update logic hasn't been running to detect it. You can follow along on our status page for when the maintenance will conclude: https://status.cdnjs.com/

I realise that this maintenance has been going on for over a month now and we still have many packages that are not being updated by the new system that is being worked on, and I can only apologise for that -- the team is working hard to get the new auto-update system up and running with all the new packages as fast as we can.

@xtuc do you think you'd be able to share insight on when you expect the new auto-update system to be live and supporting all packages on NPM/Git again?

**Reply by xtuc** [reply]
All npm packages should be auto-updating  again next week. git support will be implement soon too

**Comment by MattIPv4** [comment]
Just to close the loop here, v11 is now available on cdnjs: https://cdnjs.com/libraries/highlight.js/11.0.1
